src/resources/base.js

```diff
@@ -153,20 +153,24 @@ export default api => (
       } catch (e) {
         return Promise.reject(e);
       }
-
       try {
         const data = this.constructor.wrapJSON(this.toJSON());
-
+        let output;
         let res;
 
         if (this.id) {
           res = await api.put(`${this._url || this.constructor._url}/${this.id}`, { body: data });
+          output = this;
+        } else if (this.constructor._name === 'Rating') {
```

Comment:
Instead of changing code in resources/base to make changes specific to rating, the `rating` endpoint  should override the `save` method.

`body` is already set to `this.constructor.wrapJson(this.toJSON())`, so why do you need to set it to `this.toJSON` here again as a separate case?

Instead of setting output to `this.body`, it should be set to `this`, because `this` should have its attributes merged with the api response.